vint.go

```diff
@@ -60,3 +60,46 @@ func mustReadByte(r io.Reader) (byte, error) {
 
 	return b, err
 }
+
+func WriteVInt(w io.Writer, n int64) (error) {
+	if n >= -112 && n <= 127 {
+		_, err := w.Write([]byte{byte(int8(n))})
+		return err
+	}
+
+	len := -112
```

Comment:
I think it's allowed to shadow the top-level `len` here, but we probably don't want to do it